pkg/verify/configmap.go

```diff
@@ -8,18 +8,69 @@ import (
 
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/openpgp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
 
 	"github.com/openshift/cluster-version-operator/pkg/verify/store"
 	"github.com/openshift/cluster-version-operator/pkg/verify/store/parallel"
 	"github.com/openshift/cluster-version-operator/pkg/verify/store/sigstore"
+	"github.com/openshift/cluster-version-operator/pkg/verify/verifyutil"
 )
 
-// ReleaseAnnotationConfigMapVerifier is an annotation set on a config map in the
-// release payload to indicate that this config map controls signing for the payload.
-// Only the first config map within the payload should be used, regardless of whether
-// it has data. See NewFromConfigMapData for more.
-const ReleaseAnnotationConfigMapVerifier = "release.openshift.io/verification-config-map"
+const (
+	// ReleaseAnnotationConfigMapVerifier is an annotation set on a config map in the
+	// release payload to indicate that this config map controls signing for the payload.
+	// Only the first config map within the payload should be used, regardless of whether
+	// it has data. See NewFromConfigMapData for more.
+	ReleaseAnnotationConfigMapVerifier = "release.openshift.io/verification-config-map"
+
+	// NamespaceLabelConfigMap is the Namespace label applied to a configmap
+	// containing signatures.
+	NamespaceLabelConfigMap = "openshift-config-managed"
+
+	// ReleaseLabelConfigMap is a label applied to a configmap inside the
+	// NamespaceLabelConfigMap namespace that indicates it contains signatures
+	// for release image digests. Any binaryData key that starts with the digest
+	// is added to the list of signatures checked.
+	ReleaseLabelConfigMap = "release.openshift.io/verification-signatures"
```

Comment:
Like [the namespace][1], I don't like this being a public API with two separate definitions:

```console
$ git --no-pager grep 'verification-signatures' origin/pr/413
origin/pr/413:pkg/verify/configmap.go:	ReleaseLabelConfigMap = "release.openshift.io/verification-signatures"
origin/pr/413:pkg/verify/store/configmap/configmap.go:const ReleaseLabelConfigMap = "release.openshift.io/verification-signatures"
```

Can consumers just use the existing `pkg/verify/store/configmap` source?

[1]: https://github.com/openshift/cluster-version-operator/pull/413/files#r458951378

Reply:
Fixed in [2bde02f](https://github.com/openshift/cluster-version-operator/pull/413/commits/2bde02f1bc058d9b91c9b5cce3afb3259523b74c)